pkgs/applications/video/avidemux/default.nix

```diff
@@ -57,7 +57,11 @@ stdenv.mkDerivation rec {
 
   buildCommand = let
     wrapWith = makeWrapper: filename:
-      "${makeWrapper} ${filename} --set ADM_ROOT_DIR $out --prefix LD_LIBRARY_PATH : ${libXext}/lib";
+      ''
+        ${makeWrapper} ${filename} \
+          --set ADM_ROOT_DIR $out \
+          --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libXext (placeholder "out") ]}
```

Comment:
i think i fixed a similar issue https://github.com/NixOS/nixpkgs/pull/272491

what was happening is that cmake is installing the libs in `lib64` but the rpath points to `lib` so fixup strips the rpath because it is invalid. if you add something like.

```sh
    if [[ -d ''${!outputLib}/lib64 ]]; then
      mv ''${!outputLib}/lib64 ''${!outputLib}/lib
      ln -s lib ''${!outputLib}/lib64
    fi
```

just before `fixupPhase` in `buildCommand` then the binaries should end up with the proper rpaths.

I validated this by just running `ldd result/bin/.*` before and after the libs are found. seems like the binaries are double wrapped tho.

Reply:
@a-n-n-a-l-e-e I'm not sure I understood what your suggestion tries to fix - I tested the package now and it builds and runs fine for me, also the Nix code is clean IMO. The only issue is that the binaries are wrapped twice, as mentioned in my other comment.

Reply:
my suggestion removes having to add `(placeholder "out")` to the `LD_LIBRARY_PATH` as the RPATH will not get stripped by the fixupPhase.

Reply:
without my suggestion, `patchelf --print-rpath result-orig/bin/..*` shows:
```console
/nix/store/2xaviyb2mqfv3hcrxpf5919sa41xjsm0-libXext-1.3.5/lib:/nix/store/mp9g48br2nwn1b621np935sp915f3cr9-libXv-1.0.12/lib:/nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib:/nix/store/4zp05qkzh9s3c3k7pp9d36vhiz3x8h9l-gcc-13.2.0-lib/lib
/nix/store/bsjd4g5x76gsc0dxp2v420dz6134biyn-qtbase-5.15.12/lib:/nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib:/nix/store/4zp05qkzh9s3c3k7pp9d36vhiz3x8h9l-gcc-13.2.0-lib/lib
/nix/store/i0jkvikpn3rp3n80hw06fsw06n1v8w81-libX11-1.8.7/lib:/nix/store/2xaviyb2mqfv3hcrxpf5919sa41xjsm0-libXext-1.3.5/lib:/nix/store/cj064qzmnk2pqc3rsw3rj3g27anlvd85-libGL-1.7.0/lib:/nix/store/bsjd4g5x76gsc0dxp2v420dz6134biyn-qtbase-5.15.12/lib:/nix/store/mp9g48br2nwn1b621np935sp915f3cr9-libXv-1.0.12/lib:/nix/store/76qvjpwhzrwrkl5fz1z4r9qn2jz6x6ca-glu-9.0.3/lib:/nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib:/nix/store/4zp05qkzh9s3c3k7pp9d36vhiz3x8h9l-gcc-13.2.0-lib/lib
```

with my suggestion  `patchelf --print-rpath result-mod/bin/..*` shows `/nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib` is listed in the RPATH
```console
/nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib:/nix/store/2xaviyb2mqfv3hcrxpf5919sa41xjsm0-libXext-1.3.5/lib:/nix/store/mp9g48br2nwn1b621np935sp915f3cr9-libXv-1.0.12/lib:/nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib:/nix/store/4zp05qkzh9s3c3k7pp9d36vhiz3x8h9l-gcc-13.2.0-lib/lib
/nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib:/nix/store/bsjd4g5x76gsc0dxp2v420dz6134biyn-qtbase-5.15.12/lib:/nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib:/nix/store/4zp05qkzh9s3c3k7pp9d36vhiz3x8h9l-gcc-13.2.0-lib/lib
/nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib:/nix/store/i0jkvikpn3rp3n80hw06fsw06n1v8w81-libX11-1.8.7/lib:/nix/store/2xaviyb2mqfv3hcrxpf5919sa41xjsm0-libXext-1.3.5/lib:/nix/store/cj064qzmnk2pqc3rsw3rj3g27anlvd85-libGL-1.7.0/lib:/nix/store/bsjd4g5x76gsc0dxp2v420dz6134biyn-qtbase-5.15.12/lib:/nix/store/mp9g48br2nwn1b621np935sp915f3cr9-libXv-1.0.12/lib:/nix/store/76qvjpwhzrwrkl5fz1z4r9qn2jz6x6ca-glu-9.0.3/lib:/nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib:/nix/store/4zp05qkzh9s3c3k7pp9d36vhiz3x8h9l-gcc-13.2.0-lib/lib
```

and if you run `ldd`
<details>
<summary>ldd result-orig/bin/..*</summary>
<pre>
result-orig/bin/..avidemux3_cli-wrapped-wrapped:
	linux-vdso.so.1 (0x00007ffd64df4000)
	libADM_coreVideoCodec6.so => not found
	libADM_coreVDPAU6.so => not found
	libADM_coreLibVA6.so => not found
	libADM_coreLibVAEnc6.so => not found
	libADM6avcodec.so.58 => not found
	libADM6avutil.so.56 => not found
	libADM_coreJobs.so => not found
	libADM_core6.so => not found
	libADM_audioParser6.so => not found
	libADM_coreAudio6.so => not found
	libADM_coreAudioFilterAPI6.so => not found
	libADM_coreAudioDevice6.so => not found
	libADM_coreAudioEncoder6.so => not found
	libADM_coreDemuxer6.so => not found
	libADM_coreImage6.so => not found
	libADM_coreMuxer6.so => not found
	libADM_coreUI6.so => not found
	libADM_coreUtils6.so => not found
	libADM_coreSocket6.so => not found
	libADM_coreVideoEncoder6.so => not found
	libADM_coreVideoFilter6.so => not found
	libADM_UI_Cli6.so => not found
	libpthread.so.0 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libpthread.so.0 (0x00007f2f36a9c000)
	libADM_coreScript.so => not found
	libADM_render6_cli.so => not found
	libXv.so.1 => /nix/store/mp9g48br2nwn1b621np935sp915f3cr9-libXv-1.0.12/lib/libXv.so.1 (0x00007f2f36a92000)
	libXext.so.6 => /nix/store/2xaviyb2mqfv3hcrxpf5919sa41xjsm0-libXext-1.3.5/lib/libXext.so.6 (0x00007f2f36a7d000)
	libstdc++.so.6 => /nix/store/4zp05qkzh9s3c3k7pp9d36vhiz3x8h9l-gcc-13.2.0-lib/lib/libstdc++.so.6 (0x00007f2f36800000)
	libm.so.6 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libm.so.6 (0x00007f2f3671e000)
	libgcc_s.so.1 => /nix/store/4zp05qkzh9s3c3k7pp9d36vhiz3x8h9l-gcc-13.2.0-lib/lib/libgcc_s.so.1 (0x00007f2f366f9000)
	libc.so.6 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libc.so.6 (0x00007f2f36510000)
	libX11.so.6 => /nix/store/i0jkvikpn3rp3n80hw06fsw06n1v8w81-libX11-1.8.7/lib/libX11.so.6 (0x00007f2f363cb000)
	/nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/ld-linux-x86-64.so.2 => /lib64/ld-linux-x86-64.so.2 (0x00007f2f36ace000)
	libxcb.so.1 => /nix/store/hbksywp6j5zabm9j5yp8in1l7h524w7b-libxcb-1.16/lib/libxcb.so.1 (0x00007f2f363a0000)
	libXau.so.6 => /nix/store/zwnfxc6mnhiigdm9npyw3wb4nsqb9l2y-libXau-1.0.11/lib/libXau.so.6 (0x00007f2f36a76000)
	libXdmcp.so.6 => /nix/store/nbz9fjsrmrjkmda5r3spn43h49jcc4pi-libXdmcp-1.1.4/lib/libXdmcp.so.6 (0x00007f2f36a6c000)
result-orig/bin/..avidemux3_jobs_qt5-wrapped-wrapped:
	linux-vdso.so.1 (0x00007fffa3947000)
	libADM_coreSocket6.so => not found
	libQt5Widgets.so.5 => /nix/store/bsjd4g5x76gsc0dxp2v420dz6134biyn-qtbase-5.15.12/lib/libQt5Widgets.so.5 (0x00007f3b1dc00000)
	libpthread.so.0 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libpthread.so.0 (0x00007f3b1e395000)
	libADM_coreJobs.so => not found
	libADM_coreUtils6.so => not found
	libADM_core6.so => not found
	libstdc++.so.6 => /nix/store/4zp05qkzh9s3c3k7pp9d36vhiz3x8h9l-gcc-13.2.0-lib/lib/libstdc++.so.6 (0x00007f3b1d800000)
	libm.so.6 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libm.so.6 (0x00007f3b1db1e000)
	libQt5Gui.so.5 => /nix/store/bsjd4g5x76gsc0dxp2v420dz6134biyn-qtbase-5.15.12/lib/libQt5Gui.so.5 (0x00007f3b1d000000)
	libQt5Core.so.5 => /nix/store/bsjd4g5x76gsc0dxp2v420dz6134biyn-qtbase-5.15.12/lib/libQt5Core.so.5 (0x00007f3b1ca00000)
	libgcc_s.so.1 => /nix/store/4zp05qkzh9s3c3k7pp9d36vhiz3x8h9l-gcc-13.2.0-lib/lib/libgcc_s.so.1 (0x00007f3b1e36c000)
	libc.so.6 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libc.so.6 (0x00007f3b1c817000)
	libGL.so.1 => /nix/store/cj064qzmnk2pqc3rsw3rj3g27anlvd85-libGL-1.7.0/lib/libGL.so.1 (0x00007f3b1e2de000)
	/nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/ld-linux-x86-64.so.2 => /lib64/ld-linux-x86-64.so.2 (0x00007f3b1e3bf000)
	libpng16.so.16 => /nix/store/fsih8dis3r6j53pggqpwf72f4i5l9zh8-libpng-apng-1.6.40/lib/libpng16.so.16 (0x00007f3b1dae5000)
	libz.so.1 => /nix/store/mrzfvc2gjy2mia0k4vrb95f2iff6xcxm-zlib-1.3/lib/libz.so.1 (0x00007f3b1e2be000)
	libharfbuzz.so.0 => /nix/store/mg472bbww49f49xkcflb2b22jqjghg9h-harfbuzz-7.3.0/lib/libharfbuzz.so.0 (0x00007f3b1d6e4000)
	libicui18n.so.73 => /nix/store/dkmcrqhpyfpwabzzzn69jbavl7cjy8mm-icu4c-73.2/lib/libicui18n.so.73 (0x00007f3b1c400000)
	libicuuc.so.73 => /nix/store/dkmcrqhpyfpwabzzzn69jbavl7cjy8mm-icu4c-73.2/lib/libicuuc.so.73 (0x00007f3b1c000000)
	libicudata.so.73 => /nix/store/dkmcrqhpyfpwabzzzn69jbavl7cjy8mm-icu4c-73.2/lib/libicudata.so.73 (0x00007f3b1a000000)
	libpcre2-16.so.0 => /nix/store/r9ncz1y356780yaddaz1z888ayyvh4aa-pcre2-10.42/lib/libpcre2-16.so.0 (0x00007f3b1c788000)
	libzstd.so.1 => /nix/store/5ycxpifq1n1rs6dzhrmgbq7b862mybpp-zstd-1.5.5/lib/libzstd.so.1 (0x00007f3b1c330000)
	libgthread-2.0.so.0 => /nix/store/kl4j6fjadg3lqaqv8r9p1lsjrwmi4cz8-glib-2.78.3/lib/libgthread-2.0.so.0 (0x00007f3b1dae0000)
	libglib-2.0.so.0 => /nix/store/kl4j6fjadg3lqaqv8r9p1lsjrwmi4cz8-glib-2.78.3/lib/libglib-2.0.so.0 (0x00007f3b1beb8000)
	libGLX.so.0 => /nix/store/dwqpi8lys6lqblfqfdhkg5vj6sw12w42-libglvnd-1.7.0/lib/libGLX.so.0 (0x00007f3b1daac000)
	libX11.so.6 => /nix/store/i0jkvikpn3rp3n80hw06fsw06n1v8w81-libX11-1.8.7/lib/libX11.so.6 (0x00007f3b19ebb000)
	libXext.so.6 => /nix/store/2xaviyb2mqfv3hcrxpf5919sa41xjsm0-libXext-1.3.5/lib/libXext.so.6 (0x00007f3b1da97000)
	libGLdispatch.so.0 => /nix/store/dwqpi8lys6lqblfqfdhkg5vj6sw12w42-libglvnd-1.7.0/lib/libGLdispatch.so.0 (0x00007f3b1c277000)
	libfreetype.so.6 => /nix/store/9d2760s3khsi0hq1n6907jsmqyl73h31-freetype-2.13.2/lib/libfreetype.so.6 (0x00007f3b19ded000)
	libgraphite2.so.3 => /nix/store/9p51k9xkm13myf1kccbp7z4ndx6qiil4-graphite2-1.3.14/lib/libgraphite2.so.3 (0x00007f3b1da6e000)
	libpcre2-8.so.0 => /nix/store/r9ncz1y356780yaddaz1z888ayyvh4aa-pcre2-10.42/lib/libpcre2-8.so.0 (0x00007f3b19d50000)
	libxcb.so.1 => /nix/store/hbksywp6j5zabm9j5yp8in1l7h524w7b-libxcb-1.16/lib/libxcb.so.1 (0x00007f3b1d6b9000)
	libbz2.so.1 => /nix/store/ya47qdk3gvxhb54za3sd0jbcq7dqqqgb-bzip2-1.0.8/lib/libbz2.so.1 (0x00007f3b1d6a6000)
	libbrotlidec.so.1 => /nix/store/gs82nfl7l9ps9wn91sq1xylk7i0ad1kw-brotli-1.1.0-lib/lib/libbrotlidec.so.1 (0x00007f3b1d698000)
	libXau.so.6 => /nix/store/zwnfxc6mnhiigdm9npyw3wb4nsqb9l2y-libXau-1.0.11/lib/libXau.so.6 (0x00007f3b1da67000)
	libXdmcp.so.6 => /nix/store/nbz9fjsrmrjkmda5r3spn43h49jcc4pi-libXdmcp-1.1.4/lib/libXdmcp.so.6 (0x00007f3b1d690000)
	libbrotlicommon.so.1 => /nix/store/gs82nfl7l9ps9wn91sq1xylk7i0ad1kw-brotli-1.1.0-lib/lib/libbrotlicommon.so.1 (0x00007f3b1cfdd000)
result-orig/bin/..avidemux3_qt5-wrapped-wrapped:
	linux-vdso.so.1 (0x00007ffddd940000)
	libADM_coreVideoCodec6.so => not found
	libADM_coreVDPAU6.so => not found
	libADM_coreLibVA6.so => not found
	libADM_coreLibVAEnc6.so => not found
	libADM6avcodec.so.58 => not found
	libADM6avutil.so.56 => not found
	libADM_coreJobs.so => not found
	libADM_core6.so => not found
	libADM_audioParser6.so => not found
	libADM_coreAudio6.so => not found
	libADM_coreAudioFilterAPI6.so => not found
	libADM_coreAudioDevice6.so => not found
	libADM_coreAudioEncoder6.so => not found
	libADM_coreDemuxer6.so => not found
	libADM_coreImage6.so => not found
	libADM_coreMuxer6.so => not found
	libADM_coreUI6.so => not found
	libADM_coreUtils6.so => not found
	libADM_coreSocket6.so => not found
	libADM_coreVideoEncoder6.so => not found
	libADM_coreVideoFilter6.so => not found
	libADM_UIQT56.so => not found
	libQt5Network.so.5 => /nix/store/bsjd4g5x76gsc0dxp2v420dz6134biyn-qtbase-5.15.12/lib/libQt5Network.so.5 (0x00007f9a64b21000)
	libpthread.so.0 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libpthread.so.0 (0x00007f9a64b1a000)
	libstdc++.so.6 => /nix/store/4zp05qkzh9s3c3k7pp9d36vhiz3x8h9l-gcc-13.2.0-lib/lib/libstdc++.so.6 (0x00007f9a64800000)
	libX11.so.6 => /nix/store/i0jkvikpn3rp3n80hw06fsw06n1v8w81-libX11-1.8.7/lib/libX11.so.6 (0x00007f9a646bb000)
	libm.so.6 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libm.so.6 (0x00007f9a645d9000)
	libADM_coreScript.so => not found
	libADM_render6_QT5.so => not found
	libADM_openGLQT56.so => not found
	libQt5Widgets.so.5 => /nix/store/bsjd4g5x76gsc0dxp2v420dz6134biyn-qtbase-5.15.12/lib/libQt5Widgets.so.5 (0x00007f9a63e00000)
	libQt5Gui.so.5 => /nix/store/bsjd4g5x76gsc0dxp2v420dz6134biyn-qtbase-5.15.12/lib/libQt5Gui.so.5 (0x00007f9a63600000)
	libGL.so.1 => /nix/store/cj064qzmnk2pqc3rsw3rj3g27anlvd85-libGL-1.7.0/lib/libGL.so.1 (0x00007f9a64a8a000)
	libGLU.so.1 => /nix/store/76qvjpwhzrwrkl5fz1z4r9qn2jz6x6ca-glu-9.0.3/lib/libGLU.so.1 (0x00007f9a64564000)
	libXv.so.1 => /nix/store/mp9g48br2nwn1b621np935sp915f3cr9-libXv-1.0.12/lib/libXv.so.1 (0x00007f9a64a80000)
	libXext.so.6 => /nix/store/2xaviyb2mqfv3hcrxpf5919sa41xjsm0-libXext-1.3.5/lib/libXext.so.6 (0x00007f9a64a6b000)
	libQt5Core.so.5 => /nix/store/bsjd4g5x76gsc0dxp2v420dz6134biyn-qtbase-5.15.12/lib/libQt5Core.so.5 (0x00007f9a63000000)
	libgcc_s.so.1 => /nix/store/4zp05qkzh9s3c3k7pp9d36vhiz3x8h9l-gcc-13.2.0-lib/lib/libgcc_s.so.1 (0x00007f9a6453f000)
	libc.so.6 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libc.so.6 (0x00007f9a62e17000)
	libz.so.1 => /nix/store/mrzfvc2gjy2mia0k4vrb95f2iff6xcxm-zlib-1.3/lib/libz.so.1 (0x00007f9a64521000)
	libgssapi_krb5.so.2 => /nix/store/ynac0qi3qnpb35p72may7vb3697bapad-libkrb5-1.21.2/lib/libgssapi_krb5.so.2 (0x00007f9a644cd000)
	libssl.so.3 => /nix/store/l65pk85mh54jd7ixyf2qkq7gw0rsczky-openssl-3.0.12/lib/libssl.so.3 (0x00007f9a63d52000)
	libcrypto.so.3 => /nix/store/l65pk85mh54jd7ixyf2qkq7gw0rsczky-openssl-3.0.12/lib/libcrypto.so.3 (0x00007f9a62800000)
	/nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/ld-linux-x86-64.so.2 => /lib64/ld-linux-x86-64.so.2 (0x00007f9a64cf8000)
	libxcb.so.1 => /nix/store/hbksywp6j5zabm9j5yp8in1l7h524w7b-libxcb-1.16/lib/libxcb.so.1 (0x00007f9a63d27000)
	libpng16.so.16 => /nix/store/fsih8dis3r6j53pggqpwf72f4i5l9zh8-libpng-apng-1.6.40/lib/libpng16.so.16 (0x00007f9a63cee000)
	libharfbuzz.so.0 => /nix/store/mg472bbww49f49xkcflb2b22jqjghg9h-harfbuzz-7.3.0/lib/libharfbuzz.so.0 (0x00007f9a62cfb000)
	libGLX.so.0 => /nix/store/dwqpi8lys6lqblfqfdhkg5vj6sw12w42-libglvnd-1.7.0/lib/libGLX.so.0 (0x00007f9a63cba000)
	libGLdispatch.so.0 => /nix/store/dwqpi8lys6lqblfqfdhkg5vj6sw12w42-libglvnd-1.7.0/lib/libGLdispatch.so.0 (0x00007f9a62747000)
	libOpenGL.so.0 => /nix/store/cj064qzmnk2pqc3rsw3rj3g27anlvd85-libGL-1.7.0/lib/libOpenGL.so.0 (0x00007f9a63c8d000)
	libicui18n.so.73 => /nix/store/dkmcrqhpyfpwabzzzn69jbavl7cjy8mm-icu4c-73.2/lib/libicui18n.so.73 (0x00007f9a62200000)
	libicuuc.so.73 => /nix/store/dkmcrqhpyfpwabzzzn69jbavl7cjy8mm-icu4c-73.2/lib/libicuuc.so.73 (0x00007f9a61e00000)
	libicudata.so.73 => /nix/store/dkmcrqhpyfpwabzzzn69jbavl7cjy8mm-icu4c-73.2/lib/libicudata.so.73 (0x00007f9a5fe00000)
	libpcre2-16.so.0 => /nix/store/r9ncz1y356780yaddaz1z888ayyvh4aa-pcre2-10.42/lib/libpcre2-16.so.0 (0x00007f9a626b8000)
	libzstd.so.1 => /nix/store/5ycxpifq1n1rs6dzhrmgbq7b862mybpp-zstd-1.5.5/lib/libzstd.so.1 (0x00007f9a625e8000)
	libgthread-2.0.so.0 => /nix/store/kl4j6fjadg3lqaqv8r9p1lsjrwmi4cz8-glib-2.78.3/lib/libgthread-2.0.so.0 (0x00007f9a644c8000)
	libglib-2.0.so.0 => /nix/store/kl4j6fjadg3lqaqv8r9p1lsjrwmi4cz8-glib-2.78.3/lib/libglib-2.0.so.0 (0x00007f9a620b8000)
	libkrb5.so.3 => /nix/store/ynac0qi3qnpb35p72may7vb3697bapad-libkrb5-1.21.2/lib/libkrb5.so.3 (0x00007f9a61d29000)
	libk5crypto.so.3 => /nix/store/ynac0qi3qnpb35p72may7vb3697bapad-libkrb5-1.21.2/lib/libk5crypto.so.3 (0x00007f9a635d1000)
	libcom_err.so.3 => /nix/store/ynac0qi3qnpb35p72may7vb3697bapad-libkrb5-1.21.2/lib/libcom_err.so.3 (0x00007f9a644bf000)
	libkrb5support.so.0 => /nix/store/ynac0qi3qnpb35p72may7vb3697bapad-libkrb5-1.21.2/lib/libkrb5support.so.0 (0x00007f9a63c7f000)
	libkeyutils.so.1 => /nix/store/i28b60ahqiwfpaqicxdlpvvh63j3s05g-keyutils-1.6.3-lib/lib/libkeyutils.so.1 (0x00007f9a644b8000)
	libresolv.so.2 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libresolv.so.2 (0x00007f9a62cea000)
	libdl.so.2 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libdl.so.2 (0x00007f9a635ca000)
	libXau.so.6 => /nix/store/zwnfxc6mnhiigdm9npyw3wb4nsqb9l2y-libXau-1.0.11/lib/libXau.so.6 (0x00007f9a635c5000)
	libXdmcp.so.6 => /nix/store/nbz9fjsrmrjkmda5r3spn43h49jcc4pi-libXdmcp-1.1.4/lib/libXdmcp.so.6 (0x00007f9a62ce2000)
	libfreetype.so.6 => /nix/store/9d2760s3khsi0hq1n6907jsmqyl73h31-freetype-2.13.2/lib/libfreetype.so.6 (0x00007f9a5fd32000)
	libgraphite2.so.3 => /nix/store/9p51k9xkm13myf1kccbp7z4ndx6qiil4-graphite2-1.3.14/lib/libgraphite2.so.3 (0x00007f9a62cb7000)
	libpcre2-8.so.0 => /nix/store/r9ncz1y356780yaddaz1z888ayyvh4aa-pcre2-10.42/lib/libpcre2-8.so.0 (0x00007f9a6201b000)
	libbz2.so.1 => /nix/store/ya47qdk3gvxhb54za3sd0jbcq7dqqqgb-bzip2-1.0.8/lib/libbz2.so.1 (0x00007f9a62ca4000)
	libbrotlidec.so.1 => /nix/store/gs82nfl7l9ps9wn91sq1xylk7i0ad1kw-brotli-1.1.0-lib/lib/libbrotlidec.so.1 (0x00007f9a62c96000)
	libbrotlicommon.so.1 => /nix/store/gs82nfl7l9ps9wn91sq1xylk7i0ad1kw-brotli-1.1.0-lib/lib/libbrotlicommon.so.1 (0x00007f9a625c5000)
</pre>
</details>

<details>
<summary>ldd result-mod/bin/..*</summary>
<pre>
result-mod/bin/..avidemux3_cli-wrapped-wrapped:
	linux-vdso.so.1 (0x00007fff591b2000)
	libADM_coreVideoCodec6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreVideoCodec6.so (0x00007fcbe0e86000)
	libADM_coreVDPAU6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreVDPAU6.so (0x00007fcbe0e7c000)
	libADM_coreLibVA6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreLibVA6.so (0x00007fcbe0e6e000)
	libADM_coreLibVAEnc6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreLibVAEnc6.so (0x00007fcbe0e68000)
	libADM6avcodec.so.58 => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM6avcodec.so.58 (0x00007fcbe0200000)
	libADM6avutil.so.56 => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM6avutil.so.56 (0x00007fcbdfe00000)
	libADM_coreJobs.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreJobs.so (0x00007fcbe0e58000)
	libADM_core6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_core6.so (0x00007fcbe0e42000)
	libADM_audioParser6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_audioParser6.so (0x00007fcbe0e38000)
	libADM_coreAudio6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreAudio6.so (0x00007fcbe0dfd000)
	libADM_coreAudioFilterAPI6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreAudioFilterAPI6.so (0x00007fcbe0df8000)
	libADM_coreAudioDevice6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreAudioDevice6.so (0x00007fcbe0dee000)
	libADM_coreAudioEncoder6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreAudioEncoder6.so (0x00007fcbe0de8000)
	libADM_coreDemuxer6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreDemuxer6.so (0x00007fcbe0de1000)
	libADM_coreImage6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreImage6.so (0x00007fcbe0dc9000)
	libADM_coreMuxer6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreMuxer6.so (0x00007fcbe0dbe000)
	libADM_coreUI6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreUI6.so (0x00007fcbe0da6000)
	libADM_coreUtils6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreUtils6.so (0x00007fcbe0138000)
	libADM_coreSocket6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreSocket6.so (0x00007fcbe0d9f000)
	libADM_coreVideoEncoder6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreVideoEncoder6.so (0x00007fcbe0d8e000)
	libADM_coreVideoFilter6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreVideoFilter6.so (0x00007fcbe0d84000)
	libADM_UI_Cli6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_UI_Cli6.so (0x00007fcbe0d7f000)
	libpthread.so.0 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libpthread.so.0 (0x00007fcbe0d7a000)
	libADM_coreScript.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreScript.so (0x00007fcbe0d73000)
	libADM_render6_cli.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_render6_cli.so (0x00007fcbe0d69000)
	libXv.so.1 => /nix/store/mp9g48br2nwn1b621np935sp915f3cr9-libXv-1.0.12/lib/libXv.so.1 (0x00007fcbe0130000)
	libXext.so.6 => /nix/store/2xaviyb2mqfv3hcrxpf5919sa41xjsm0-libXext-1.3.5/lib/libXext.so.6 (0x00007fcbe011b000)
	libstdc++.so.6 => /nix/store/4zp05qkzh9s3c3k7pp9d36vhiz3x8h9l-gcc-13.2.0-lib/lib/libstdc++.so.6 (0x00007fcbdfa00000)
	libm.so.6 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libm.so.6 (0x00007fcbdfd1e000)
	libgcc_s.so.1 => /nix/store/4zp05qkzh9s3c3k7pp9d36vhiz3x8h9l-gcc-13.2.0-lib/lib/libgcc_s.so.1 (0x00007fcbe00f6000)
	libc.so.6 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libc.so.6 (0x00007fcbdf817000)
	libva.so.2 => /nix/store/2p2di92jwcwf3fsav8zdngfn9hmdvinb-libva-2.20.0/lib/libva.so.2 (0x00007fcbe00c5000)
	libva-x11.so.2 => /nix/store/2p2di92jwcwf3fsav8zdngfn9hmdvinb-libva-2.20.0/lib/libva-x11.so.2 (0x00007fcbe00bd000)
	libz.so.1 => /nix/store/mrzfvc2gjy2mia0k4vrb95f2iff6xcxm-zlib-1.3/lib/libz.so.1 (0x00007fcbdfd00000)
	libva-drm.so.2 => /nix/store/2p2di92jwcwf3fsav8zdngfn9hmdvinb-libva-2.20.0/lib/libva-drm.so.2 (0x00007fcbe00b6000)
	libvdpau.so.1 => /nix/store/glaa27hpvbbns38dlparckgbmq3hfa1l-libvdpau-1.5/lib/libvdpau.so.1 (0x00007fcbe00af000)
	libX11.so.6 => /nix/store/i0jkvikpn3rp3n80hw06fsw06n1v8w81-libX11-1.8.7/lib/libX11.so.6 (0x00007fcbdf6d2000)
	libADM_coreSqlLight3.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreSqlLight3.so (0x00007fcbe00a2000)
	libADM6swscale.so.5 => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM6swscale.so.5 (0x00007fcbdfc69000)
	libADM6postproc.so.55 => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM6postproc.so.55 (0x00007fcbdf6a9000)
	libADM6avformat.so.58 => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM6avformat.so.58 (0x00007fcbdf62d000)
	/nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/ld-linux-x86-64.so.2 => /lib64/ld-linux-x86-64.so.2 (0x00007fcbe0e9a000)
	libdrm.so.2 => /nix/store/589gf7jf57pcmzlq5fwq4jzn3a9m9zc0-libdrm-2.4.119/lib/libdrm.so.2 (0x00007fcbdf615000)
	libXfixes.so.3 => /nix/store/7hij9gahccyi1793dgi80qqqnbj4f4bv-libXfixes-6.0.1/lib/libXfixes.so.3 (0x00007fcbdf60d000)
	libX11-xcb.so.1 => /nix/store/i0jkvikpn3rp3n80hw06fsw06n1v8w81-libX11-1.8.7/lib/libX11-xcb.so.1 (0x00007fcbdfc62000)
	libxcb.so.1 => /nix/store/hbksywp6j5zabm9j5yp8in1l7h524w7b-libxcb-1.16/lib/libxcb.so.1 (0x00007fcbdf5e2000)
	libxcb-dri3.so.0 => /nix/store/hbksywp6j5zabm9j5yp8in1l7h524w7b-libxcb-1.16/lib/libxcb-dri3.so.0 (0x00007fcbdf5dc000)
	libsqlite3.so.0 => /nix/store/izgv5wkimsmfigak4xkwvs5bjnjrlf3h-sqlite-3.44.2/lib/libsqlite3.so.0 (0x00007fcbdf47f000)
	libXau.so.6 => /nix/store/zwnfxc6mnhiigdm9npyw3wb4nsqb9l2y-libXau-1.0.11/lib/libXau.so.6 (0x00007fcbdf47a000)
	libXdmcp.so.6 => /nix/store/nbz9fjsrmrjkmda5r3spn43h49jcc4pi-libXdmcp-1.1.4/lib/libXdmcp.so.6 (0x00007fcbdf472000)
result-mod/bin/..avidemux3_jobs_qt5-wrapped-wrapped:
	linux-vdso.so.1 (0x00007ffdea7f1000)
	libADM_coreSocket6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreSocket6.so (0x00007f026f14e000)
	libQt5Widgets.so.5 => /nix/store/bsjd4g5x76gsc0dxp2v420dz6134biyn-qtbase-5.15.12/lib/libQt5Widgets.so.5 (0x00007f026ea00000)
	libpthread.so.0 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libpthread.so.0 (0x00007f026f149000)
	libADM_coreJobs.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreJobs.so (0x00007f026f13b000)
	libADM_coreUtils6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreUtils6.so (0x00007f026e938000)
	libADM_core6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_core6.so (0x00007f026f123000)
	libstdc++.so.6 => /nix/store/4zp05qkzh9s3c3k7pp9d36vhiz3x8h9l-gcc-13.2.0-lib/lib/libstdc++.so.6 (0x00007f026e600000)
	libm.so.6 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libm.so.6 (0x00007f026e51e000)
	libQt5Gui.so.5 => /nix/store/bsjd4g5x76gsc0dxp2v420dz6134biyn-qtbase-5.15.12/lib/libQt5Gui.so.5 (0x00007f026de00000)
	libQt5Core.so.5 => /nix/store/bsjd4g5x76gsc0dxp2v420dz6134biyn-qtbase-5.15.12/lib/libQt5Core.so.5 (0x00007f026d800000)
	libgcc_s.so.1 => /nix/store/4zp05qkzh9s3c3k7pp9d36vhiz3x8h9l-gcc-13.2.0-lib/lib/libgcc_s.so.1 (0x00007f026f0fc000)
	libc.so.6 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libc.so.6 (0x00007f026d617000)
	libGL.so.1 => /nix/store/cj064qzmnk2pqc3rsw3rj3g27anlvd85-libGL-1.7.0/lib/libGL.so.1 (0x00007f026e8aa000)
	libADM_coreSqlLight3.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreSqlLight3.so (0x00007f026f0ef000)
	libADM_coreUI6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreUI6.so (0x00007f026f0d5000)
	libADM6avutil.so.56 => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM6avutil.so.56 (0x00007f026d200000)
	libADM6avcodec.so.58 => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM6avcodec.so.58 (0x00007f026c600000)
	/nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/ld-linux-x86-64.so.2 => /lib64/ld-linux-x86-64.so.2 (0x00007f026f157000)
	libpng16.so.16 => /nix/store/fsih8dis3r6j53pggqpwf72f4i5l9zh8-libpng-apng-1.6.40/lib/libpng16.so.16 (0x00007f026e871000)
	libz.so.1 => /nix/store/mrzfvc2gjy2mia0k4vrb95f2iff6xcxm-zlib-1.3/lib/libz.so.1 (0x00007f026e500000)
	libharfbuzz.so.0 => /nix/store/mg472bbww49f49xkcflb2b22jqjghg9h-harfbuzz-7.3.0/lib/libharfbuzz.so.0 (0x00007f026d4fb000)
	libicui18n.so.73 => /nix/store/dkmcrqhpyfpwabzzzn69jbavl7cjy8mm-icu4c-73.2/lib/libicui18n.so.73 (0x00007f026c200000)
	libicuuc.so.73 => /nix/store/dkmcrqhpyfpwabzzzn69jbavl7cjy8mm-icu4c-73.2/lib/libicuuc.so.73 (0x00007f026be00000)
	libicudata.so.73 => /nix/store/dkmcrqhpyfpwabzzzn69jbavl7cjy8mm-icu4c-73.2/lib/libicudata.so.73 (0x00007f0269e00000)
	libpcre2-16.so.0 => /nix/store/r9ncz1y356780yaddaz1z888ayyvh4aa-pcre2-10.42/lib/libpcre2-16.so.0 (0x00007f026d171000)
	libzstd.so.1 => /nix/store/5ycxpifq1n1rs6dzhrmgbq7b862mybpp-zstd-1.5.5/lib/libzstd.so.1 (0x00007f026c130000)
	libgthread-2.0.so.0 => /nix/store/kl4j6fjadg3lqaqv8r9p1lsjrwmi4cz8-glib-2.78.3/lib/libgthread-2.0.so.0 (0x00007f026f0cc000)
	libglib-2.0.so.0 => /nix/store/kl4j6fjadg3lqaqv8r9p1lsjrwmi4cz8-glib-2.78.3/lib/libglib-2.0.so.0 (0x00007f026bcb8000)
	libGLX.so.0 => /nix/store/dwqpi8lys6lqblfqfdhkg5vj6sw12w42-libglvnd-1.7.0/lib/libGLX.so.0 (0x00007f026e4cc000)
	libX11.so.6 => /nix/store/i0jkvikpn3rp3n80hw06fsw06n1v8w81-libX11-1.8.7/lib/libX11.so.6 (0x00007f0269cbb000)
	libXext.so.6 => /nix/store/2xaviyb2mqfv3hcrxpf5919sa41xjsm0-libXext-1.3.5/lib/libXext.so.6 (0x00007f026e4b7000)
	libGLdispatch.so.0 => /nix/store/dwqpi8lys6lqblfqfdhkg5vj6sw12w42-libglvnd-1.7.0/lib/libGLdispatch.so.0 (0x00007f026c077000)
	libsqlite3.so.0 => /nix/store/izgv5wkimsmfigak4xkwvs5bjnjrlf3h-sqlite-3.44.2/lib/libsqlite3.so.0 (0x00007f0269b60000)
	libva-drm.so.2 => /nix/store/2p2di92jwcwf3fsav8zdngfn9hmdvinb-libva-2.20.0/lib/libva-drm.so.2 (0x00007f026f0c3000)
	libva.so.2 => /nix/store/2p2di92jwcwf3fsav8zdngfn9hmdvinb-libva-2.20.0/lib/libva.so.2 (0x00007f026e486000)
	libva-x11.so.2 => /nix/store/2p2di92jwcwf3fsav8zdngfn9hmdvinb-libva-2.20.0/lib/libva-x11.so.2 (0x00007f026f0bb000)
	libvdpau.so.1 => /nix/store/glaa27hpvbbns38dlparckgbmq3hfa1l-libvdpau-1.5/lib/libvdpau.so.1 (0x00007f026e86a000)
	libfreetype.so.6 => /nix/store/9d2760s3khsi0hq1n6907jsmqyl73h31-freetype-2.13.2/lib/libfreetype.so.6 (0x00007f0269a92000)
	libgraphite2.so.3 => /nix/store/9p51k9xkm13myf1kccbp7z4ndx6qiil4-graphite2-1.3.14/lib/libgraphite2.so.3 (0x00007f026ddd7000)
	libpcre2-8.so.0 => /nix/store/r9ncz1y356780yaddaz1z888ayyvh4aa-pcre2-10.42/lib/libpcre2-8.so.0 (0x00007f026c563000)
	libxcb.so.1 => /nix/store/hbksywp6j5zabm9j5yp8in1l7h524w7b-libxcb-1.16/lib/libxcb.so.1 (0x00007f026d4d0000)
	libdrm.so.2 => /nix/store/589gf7jf57pcmzlq5fwq4jzn3a9m9zc0-libdrm-2.4.119/lib/libdrm.so.2 (0x00007f026d4b8000)
	libXfixes.so.3 => /nix/store/7hij9gahccyi1793dgi80qqqnbj4f4bv-libXfixes-6.0.1/lib/libXfixes.so.3 (0x00007f026e47e000)
	libX11-xcb.so.1 => /nix/store/i0jkvikpn3rp3n80hw06fsw06n1v8w81-libX11-1.8.7/lib/libX11-xcb.so.1 (0x00007f026e863000)
	libxcb-dri3.so.0 => /nix/store/hbksywp6j5zabm9j5yp8in1l7h524w7b-libxcb-1.16/lib/libxcb-dri3.so.0 (0x00007f026ddd1000)
	libbz2.so.1 => /nix/store/ya47qdk3gvxhb54za3sd0jbcq7dqqqgb-bzip2-1.0.8/lib/libbz2.so.1 (0x00007f026d4a5000)
	libbrotlidec.so.1 => /nix/store/gs82nfl7l9ps9wn91sq1xylk7i0ad1kw-brotli-1.1.0-lib/lib/libbrotlidec.so.1 (0x00007f026c555000)
	libXau.so.6 => /nix/store/zwnfxc6mnhiigdm9npyw3wb4nsqb9l2y-libXau-1.0.11/lib/libXau.so.6 (0x00007f026ddca000)
	libXdmcp.so.6 => /nix/store/nbz9fjsrmrjkmda5r3spn43h49jcc4pi-libXdmcp-1.1.4/lib/libXdmcp.so.6 (0x00007f026d169000)
	libbrotlicommon.so.1 => /nix/store/gs82nfl7l9ps9wn91sq1xylk7i0ad1kw-brotli-1.1.0-lib/lib/libbrotlicommon.so.1 (0x00007f026c054000)
result-mod/bin/..avidemux3_qt5-wrapped-wrapped:
	linux-vdso.so.1 (0x00007fff1b330000)
	libADM_coreVideoCodec6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreVideoCodec6.so (0x00007fc9a0f10000)
	libADM_coreVDPAU6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreVDPAU6.so (0x00007fc9a0f06000)
	libADM_coreLibVA6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreLibVA6.so (0x00007fc9a0ef8000)
	libADM_coreLibVAEnc6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreLibVAEnc6.so (0x00007fc9a0ef0000)
	libADM6avcodec.so.58 => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM6avcodec.so.58 (0x00007fc9a0200000)
	libADM6avutil.so.56 => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM6avutil.so.56 (0x00007fc99fe00000)
	libADM_coreJobs.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreJobs.so (0x00007fc9a0ee2000)
	libADM_core6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_core6.so (0x00007fc9a0ecc000)
	libADM_audioParser6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_audioParser6.so (0x00007fc9a0ec0000)
	libADM_coreAudio6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreAudio6.so (0x00007fc9a0e87000)
	libADM_coreAudioFilterAPI6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreAudioFilterAPI6.so (0x00007fc9a0e82000)
	libADM_coreAudioDevice6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreAudioDevice6.so (0x00007fc9a0e78000)
	libADM_coreAudioEncoder6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreAudioEncoder6.so (0x00007fc9a0e72000)
	libADM_coreDemuxer6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreDemuxer6.so (0x00007fc9a0e69000)
	libADM_coreImage6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreImage6.so (0x00007fc9a0e53000)
	libADM_coreMuxer6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreMuxer6.so (0x00007fc9a0e48000)
	libADM_coreUI6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreUI6.so (0x00007fc9a0e30000)
	libADM_coreUtils6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreUtils6.so (0x00007fc9a0d68000)
	libADM_coreSocket6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreSocket6.so (0x00007fc9a01f7000)
	libADM_coreVideoEncoder6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreVideoEncoder6.so (0x00007fc9a01e8000)
	libADM_coreVideoFilter6.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreVideoFilter6.so (0x00007fc9a01de000)
	libADM_UIQT56.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_UIQT56.so (0x00007fc9a0199000)
	libQt5Network.so.5 => /nix/store/bsjd4g5x76gsc0dxp2v420dz6134biyn-qtbase-5.15.12/lib/libQt5Network.so.5 (0x00007fc99fc56000)
	libpthread.so.0 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libpthread.so.0 (0x00007fc9a0192000)
	libstdc++.so.6 => /nix/store/4zp05qkzh9s3c3k7pp9d36vhiz3x8h9l-gcc-13.2.0-lib/lib/libstdc++.so.6 (0x00007fc99f800000)
	libX11.so.6 => /nix/store/i0jkvikpn3rp3n80hw06fsw06n1v8w81-libX11-1.8.7/lib/libX11.so.6 (0x00007fc99fb11000)
	libm.so.6 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libm.so.6 (0x00007fc9a00b0000)
	libADM_coreScript.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreScript.so (0x00007fc9a00a9000)
	libADM_render6_QT5.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_render6_QT5.so (0x00007fc99fb01000)
	libADM_openGLQT56.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_openGLQT56.so (0x00007fc99faf5000)
	libQt5Widgets.so.5 => /nix/store/bsjd4g5x76gsc0dxp2v420dz6134biyn-qtbase-5.15.12/lib/libQt5Widgets.so.5 (0x00007fc99f000000)
	libQt5Gui.so.5 => /nix/store/bsjd4g5x76gsc0dxp2v420dz6134biyn-qtbase-5.15.12/lib/libQt5Gui.so.5 (0x00007fc99e800000)
	libGL.so.1 => /nix/store/cj064qzmnk2pqc3rsw3rj3g27anlvd85-libGL-1.7.0/lib/libGL.so.1 (0x00007fc99fa67000)
	libGLU.so.1 => /nix/store/76qvjpwhzrwrkl5fz1z4r9qn2jz6x6ca-glu-9.0.3/lib/libGLU.so.1 (0x00007fc99f78b000)
	libXv.so.1 => /nix/store/mp9g48br2nwn1b621np935sp915f3cr9-libXv-1.0.12/lib/libXv.so.1 (0x00007fc99f783000)
	libXext.so.6 => /nix/store/2xaviyb2mqfv3hcrxpf5919sa41xjsm0-libXext-1.3.5/lib/libXext.so.6 (0x00007fc99f76e000)
	libQt5Core.so.5 => /nix/store/bsjd4g5x76gsc0dxp2v420dz6134biyn-qtbase-5.15.12/lib/libQt5Core.so.5 (0x00007fc99e200000)
	libgcc_s.so.1 => /nix/store/4zp05qkzh9s3c3k7pp9d36vhiz3x8h9l-gcc-13.2.0-lib/lib/libgcc_s.so.1 (0x00007fc99f749000)
	libc.so.6 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libc.so.6 (0x00007fc99e017000)
	libva.so.2 => /nix/store/2p2di92jwcwf3fsav8zdngfn9hmdvinb-libva-2.20.0/lib/libva.so.2 (0x00007fc99f718000)
	libva-x11.so.2 => /nix/store/2p2di92jwcwf3fsav8zdngfn9hmdvinb-libva-2.20.0/lib/libva-x11.so.2 (0x00007fc99f710000)
	libz.so.1 => /nix/store/mrzfvc2gjy2mia0k4vrb95f2iff6xcxm-zlib-1.3/lib/libz.so.1 (0x00007fc99f6f2000)
	libva-drm.so.2 => /nix/store/2p2di92jwcwf3fsav8zdngfn9hmdvinb-libva-2.20.0/lib/libva-drm.so.2 (0x00007fc99fa62000)
	libvdpau.so.1 => /nix/store/glaa27hpvbbns38dlparckgbmq3hfa1l-libvdpau-1.5/lib/libvdpau.so.1 (0x00007fc99f6eb000)
	libADM_coreSqlLight3.so => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM_coreSqlLight3.so (0x00007fc99f6de000)
	libADM6swscale.so.5 => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM6swscale.so.5 (0x00007fc99ef69000)
	libADM6postproc.so.55 => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM6postproc.so.55 (0x00007fc99ef40000)
	libADM6avformat.so.58 => /nix/store/8wssaqcpr5gg4hcjns6mfgkwmgfa4fb6-avidemux-2.8.1/lib/libADM6avformat.so.58 (0x00007fc99eec4000)
	libgssapi_krb5.so.2 => /nix/store/ynac0qi3qnpb35p72may7vb3697bapad-libkrb5-1.21.2/lib/libgssapi_krb5.so.2 (0x00007fc99dfc3000)
	libssl.so.3 => /nix/store/l65pk85mh54jd7ixyf2qkq7gw0rsczky-openssl-3.0.12/lib/libssl.so.3 (0x00007fc99df15000)
	libcrypto.so.3 => /nix/store/l65pk85mh54jd7ixyf2qkq7gw0rsczky-openssl-3.0.12/lib/libcrypto.so.3 (0x00007fc99da00000)
	/nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/ld-linux-x86-64.so.2 => /lib64/ld-linux-x86-64.so.2 (0x00007fc9a0f24000)
	libxcb.so.1 => /nix/store/hbksywp6j5zabm9j5yp8in1l7h524w7b-libxcb-1.16/lib/libxcb.so.1 (0x00007fc99ee99000)
	libpng16.so.16 => /nix/store/fsih8dis3r6j53pggqpwf72f4i5l9zh8-libpng-apng-1.6.40/lib/libpng16.so.16 (0x00007fc99e7c7000)
	libharfbuzz.so.0 => /nix/store/mg472bbww49f49xkcflb2b22jqjghg9h-harfbuzz-7.3.0/lib/libharfbuzz.so.0 (0x00007fc99d8e4000)
	libGLX.so.0 => /nix/store/dwqpi8lys6lqblfqfdhkg5vj6sw12w42-libglvnd-1.7.0/lib/libGLX.so.0 (0x00007fc99dee1000)
	libGLdispatch.so.0 => /nix/store/dwqpi8lys6lqblfqfdhkg5vj6sw12w42-libglvnd-1.7.0/lib/libGLdispatch.so.0 (0x00007fc99d82b000)
	libOpenGL.so.0 => /nix/store/cj064qzmnk2pqc3rsw3rj3g27anlvd85-libGL-1.7.0/lib/libOpenGL.so.0 (0x00007fc99deb4000)
	libicui18n.so.73 => /nix/store/dkmcrqhpyfpwabzzzn69jbavl7cjy8mm-icu4c-73.2/lib/libicui18n.so.73 (0x00007fc99d400000)
	libicuuc.so.73 => /nix/store/dkmcrqhpyfpwabzzzn69jbavl7cjy8mm-icu4c-73.2/lib/libicuuc.so.73 (0x00007fc99d000000)
	libicudata.so.73 => /nix/store/dkmcrqhpyfpwabzzzn69jbavl7cjy8mm-icu4c-73.2/lib/libicudata.so.73 (0x00007fc99b000000)
	libpcre2-16.so.0 => /nix/store/r9ncz1y356780yaddaz1z888ayyvh4aa-pcre2-10.42/lib/libpcre2-16.so.0 (0x00007fc99d79c000)
	libzstd.so.1 => /nix/store/5ycxpifq1n1rs6dzhrmgbq7b862mybpp-zstd-1.5.5/lib/libzstd.so.1 (0x00007fc99d330000)
	libgthread-2.0.so.0 => /nix/store/kl4j6fjadg3lqaqv8r9p1lsjrwmi4cz8-glib-2.78.3/lib/libgthread-2.0.so.0 (0x00007fc99f6d1000)
	libglib-2.0.so.0 => /nix/store/kl4j6fjadg3lqaqv8r9p1lsjrwmi4cz8-glib-2.78.3/lib/libglib-2.0.so.0 (0x00007fc99ceb8000)
	libdrm.so.2 => /nix/store/589gf7jf57pcmzlq5fwq4jzn3a9m9zc0-libdrm-2.4.119/lib/libdrm.so.2 (0x00007fc99ee81000)
	libXfixes.so.3 => /nix/store/7hij9gahccyi1793dgi80qqqnbj4f4bv-libXfixes-6.0.1/lib/libXfixes.so.3 (0x00007fc99f6c7000)
	libX11-xcb.so.1 => /nix/store/i0jkvikpn3rp3n80hw06fsw06n1v8w81-libX11-1.8.7/lib/libX11-xcb.so.1 (0x00007fc99f6c2000)
	libxcb-dri3.so.0 => /nix/store/hbksywp6j5zabm9j5yp8in1l7h524w7b-libxcb-1.16/lib/libxcb-dri3.so.0 (0x00007fc99f6bc000)
	libsqlite3.so.0 => /nix/store/izgv5wkimsmfigak4xkwvs5bjnjrlf3h-sqlite-3.44.2/lib/libsqlite3.so.0 (0x00007fc99aea5000)
	libkrb5.so.3 => /nix/store/ynac0qi3qnpb35p72may7vb3697bapad-libkrb5-1.21.2/lib/libkrb5.so.3 (0x00007fc99d259000)
	libk5crypto.so.3 => /nix/store/ynac0qi3qnpb35p72may7vb3697bapad-libkrb5-1.21.2/lib/libk5crypto.so.3 (0x00007fc99de85000)
	libcom_err.so.3 => /nix/store/ynac0qi3qnpb35p72may7vb3697bapad-libkrb5-1.21.2/lib/libcom_err.so.3 (0x00007fc99de7e000)
	libkrb5support.so.0 => /nix/store/ynac0qi3qnpb35p72may7vb3697bapad-libkrb5-1.21.2/lib/libkrb5support.so.0 (0x00007fc99d78e000)
	libkeyutils.so.1 => /nix/store/i28b60ahqiwfpaqicxdlpvvh63j3s05g-keyutils-1.6.3-lib/lib/libkeyutils.so.1 (0x00007fc99d787000)
	libresolv.so.2 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libresolv.so.2 (0x00007fc99d776000)
	libdl.so.2 => /nix/store/p3jshbwxiwifm1py0yq544fmdyy98j8a-glibc-2.38-27/lib/libdl.so.2 (0x00007fc99d771000)
	libXau.so.6 => /nix/store/zwnfxc6mnhiigdm9npyw3wb4nsqb9l2y-libXau-1.0.11/lib/libXau.so.6 (0x00007fc99d76c000)
	libXdmcp.so.6 => /nix/store/nbz9fjsrmrjkmda5r3spn43h49jcc4pi-libXdmcp-1.1.4/lib/libXdmcp.so.6 (0x00007fc99d764000)
	libfreetype.so.6 => /nix/store/9d2760s3khsi0hq1n6907jsmqyl73h31-freetype-2.13.2/lib/libfreetype.so.6 (0x00007fc99add7000)
	libgraphite2.so.3 => /nix/store/9p51k9xkm13myf1kccbp7z4ndx6qiil4-graphite2-1.3.14/lib/libgraphite2.so.3 (0x00007fc99d230000)
	libpcre2-8.so.0 => /nix/store/r9ncz1y356780yaddaz1z888ayyvh4aa-pcre2-10.42/lib/libpcre2-8.so.0 (0x00007fc99ad3a000)
	libbz2.so.1 => /nix/store/ya47qdk3gvxhb54za3sd0jbcq7dqqqgb-bzip2-1.0.8/lib/libbz2.so.1 (0x00007fc99d751000)
	libbrotlidec.so.1 => /nix/store/gs82nfl7l9ps9wn91sq1xylk7i0ad1kw-brotli-1.1.0-lib/lib/libbrotlidec.so.1 (0x00007fc99d222000)
	libbrotlicommon.so.1 => /nix/store/gs82nfl7l9ps9wn91sq1xylk7i0ad1kw-brotli-1.1.0-lib/lib/libbrotlicommon.so.1 (0x00007fc99ce95000)
</pre>
</details>

Reply:
Your suggestion seems better to me ... since it sounds like you already have it implemented, do you want to open a PR for it and we can close this one?

Reply:
sure, though, i've never run the program before -- i just recognized the issue.
https://github.com/NixOS/nixpkgs/pull/281173